files/build_templates/docker_image_ctl.j2

```diff
@@ -383,6 +383,7 @@ start() {
 {%- if sonic_asic_platform == "mellanox" %}
 {%- if docker_container_name == "syncd" %}
         -v /var/log/mellanox/sniffer:/var/log/mellanox/sniffer:rw \
+        -v /var/log/mellanox/sdk-dumps:/var/log/mellanox/sdk-dumps:rw \
```

Comment:
>/var/log/mellano [](http://example.com/codeflow?start=11&length=16)

Is it better to mount `/var/log/mellanox/` so you have every subfolder?

Reply:
While mounting the mellanox folder would mount every subfolder, it would not create the subfolder.
It won't be a problem for the sdk-dumps since I am making sure the folder is created later on, the sniffer folder is not being created when we remove it from here.

Reply:
@keboliu Could you help check "the sniffer folder is not being created when we remove it from here?"

Reply:
@qiluo-msft sniffer function need "/var/log/mellanox/sniffer"  folder to be created beforehand,  if we remove the "sniffer" folder here, the sniffer function can not be enabled properly.

Reply:
@keboliu Could you relax this limitation? I would like unify all the directories to just `/var/log/mellanox/`

Reply:
@qiluo-msft it should be doable for the sniffer subfolder, I see some other subfolders need to check internally and get back

Reply:
@qiluo-msft  please check PR:  https://github.com/Azure/sonic-buildimage/pull/7830  and https://github.com/Azure/sonic-utilities/pull/1659